### ISSUE TITLE
Missing some flags.

### ISSUE THREAD

**Issue by MastaCoder** [CigarProject/Cigar#434]
MultiOgar has some flags sent by the user system set in place for the chat. (And cell I think?) We should add these flags to the chat of Cigar.

I can do this in 2 weeks (for my break, they pushed it back for some stupid reason), or someone can help already 👍

**Comment by Luka967** [comment]
Will help but I need to understand the main_out structure first... xD

**Comment by MastaCoder** [comment]
@Luka967 Yeah I've been trying to learn it too 👍 I know most of the updateNodes section, trying to figure out how rendering really works and how I can improve it. I've got a long way to go, but closer than before.

Good luck to you as well xD

**Comment by agariotips** [comment]
```JS
// flags
    var flags = 0;
    if (this.sender == null)
        flags = 0x80;           // server message
    else if (this.sender.userRole == UserRoleEnum.ADMIN)
        flags = 0x40;           // admin message
    else if (this.sender.userRole == UserRoleEnum.MODER)
        flags = 0x20;           // moder message
```
these are the flags for admin/moderator/server messages from MultiOgar.

**Comment by MastaCoder** [comment]
Yep, I know. @agariotips Just trying to figure out how the heck they work.
